data/README.md

```diff
@@ -67,6 +76,10 @@ instance, they should be in the same format as the files below.
   where each row represents a gene set, except this file does not include a
   description column, which is the second column in the GMT file format.
 
+  As we mentioned in the description of *sample_signature_activity.txt*,
+  depending on your preference of the signatures' order shown online, you may
+  need to pad some signatures' names.
```

Comment:
Hmmm, I think this sentence is a little hard to follow. Maybe just changing the order and words a bit to something like "As we mentioned in the description of *sample_signature_activity.txt*, you may
need to pad some signatures' names depending on your preferred order of the signatures in the web interface."

Reply:
@rzelayafavila: Yes, I agree this sentence reads a little awkward. @mhuyck suggested another alternative which I think is great. If you also agree, I will change to his version.

Reply:
Sure, sounds good